ci/nixpkgs-vet.nix

```
@@ -16,6 +16,9 @@ let
       fileset = (gitTracked path);
       root = path;
     };
+
+  base' = filtered base;
+  head' = filtered head;
```

Nice!

Is it worth naming these more explicitly, e.g:

```suggestion
  filteredBase = filtered base;
  filteredHead = filtered head;
```

```suggestion
  baseFiltered = filtered base;
  headFiltered = filtered head;
```

?